pkg/exchange/ftx/rest_order_request.go

```diff
@@ -1,153 +1,6 @@
 package ftx
 
-import (
-	"context"
-	"encoding/json"
-	"fmt"
-	"strconv"
-	"time"
-	"github.com/c9s/bbgo/pkg/fixedpoint"
-)
-
 type orderRequest struct {
 	*restRequest
```

Comment:
remove this file? ordeRequest definition here seems redundant

Reply:
yeah